src/DevChatter.DevStreams.Core/Helpers/TimeZoneHelper.cs

```diff
@@ -0,0 +1,34 @@
+﻿using NodaTime;
+using System;
+
+namespace DevChatter.DevStreams.Core.Helpers
+{
+    public static class TimeZoneHelper
+    {
+        public static (int adjustDayOfWeek, LocalTime localTime) ConvertLocalTimeToDifferentTimeZone(LocalTime fromTime, string fromZoneId, string toZoneId)
```

Comment:
Any reason not to wire this up as an extension method?

Reply:
No reason, it just didn't evolve that way on-stream :-)